AppCenterIdentity/AppCenterIdentity/Internals/Storage/MSKeychainAuthTokenStorage.h

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSAuthTokenStorage.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSKeychainAuthTokenStorage : NSObject <MSAuthTokenStorage>
+
+@property(nullable, nonatomic) NSMutableArray<MSAuthTokenInfo *> *authTokensHistoryState;
```

Comment:
Add a doc for this property. Why does the name contain `State`?

Reply:
> Add a doc for this property.
Sure, added.

> Why does the name contain State?
It seems `State` suffix is too confusing and actually doesn't reflect the purpose of the property so I removed it. Thanks.